src/app/program/program.component.ts

```diff
@@ -69,7 +70,8 @@ export class ProgramComponent implements OnInit, AfterViewInit {
         }).result;
     }
 
-    addProgram = () => {
+    // nothing is just there, that no error will be thrown
+    addProgram(nothing: string = "") {
```

Comment:
rename to '\_nothing' or simply '\_'

Reply:
resolved by removing unused param and adjusting tests accordingly